### DISCUSSION TITLE
Possible to rename tailwind.css?

### DISCUSSION THREAD

**Discussion by ryanaltvater** [tailwindlabs/tailwindcss#7074] [Help] [answered]
Is it possible to rename `tailwind.css` to something like `styles.css/scss`, or is it a requirement that it be named `tailwind.css`? I've seen in documentation the `tailwind.css` file referenced as `input.css`, `output.css`, and `main.css` when demoing some of the configuration steps and setup. Personally, I'd like to name it `styles.scss` with the hope that I can still write custom SCSS or import other SCSS partials, when needed. Otherwise, I think I'd have to have 2 separate files `tailwind.css` and `styles.scss`. Thanks for the help in advance!

**Comment by adamwathan** [comment, marked as answer]
Hey! You can name it whatever you want, just make sure you pass that file name to whatever build tool you are using to process your CSS so it actually processes that file of course.

**Reply by ryanaltvater** [reply]
Hmm, okay. I’ll have to give it a shot again because within the nuxt.config.js file where you call tailwind.css for compilation I changed it there and things broke.

**Reply by ryanaltvater** [reply]
Okay, I must've been doing something wrong initially because now it seems to be working for me. Thank you @adamwathan
